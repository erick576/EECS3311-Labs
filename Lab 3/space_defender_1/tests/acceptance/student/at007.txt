play(10, 5, 500, 5)
