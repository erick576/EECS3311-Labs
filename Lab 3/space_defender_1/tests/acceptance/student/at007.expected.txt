Type Error: specification of command executions is not type-correct
play(10, 5, 500, 5) does not conform to declaration play(row: MAX_ROW = 3 .. 10 ; column: MAX_COLUMN = 5 .. 30 ; player_mov: PLAYER_MOV = 1 .. 40 ; project_mov: PROJECT_MOV = 1 .. 5)
